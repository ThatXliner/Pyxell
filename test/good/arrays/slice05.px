
a = [5, 6, 7, 8]
print a[2:-4:-1]
print a[-3::-1]
print a[:0:-1]
print a[::-3]
print (a*100)[-100::-21].length
