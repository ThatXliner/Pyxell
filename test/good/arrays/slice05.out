[7, 6]
[6, 5]
[8, 7, 6]
[8, 5]
15
